### PR TITLE
Enable clinical decision support for all facilities in Ethiopia

### PR DESCRIPTION
**Pull request by siddh1004**
https://app.shortcut.com/simpledotorg/story/10567/enable-clinical-decision-support-for-all-facilities-in-ethiopia

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt [hunk 1]
```diff
@@ -96,7 +96,9 @@ class PatientSummaryEffectHandler @AssistedInject constructor(
           .observeOn(schedulersProvider.io())
           .map {
             val currentFacilityId = currentFacility.get().uuid
-            CDSSPilotStatusChecked(isPilotEnabledForFacility = cdssPilotFacilities.get().contains(currentFacilityId))
+            CDSSPilotStatusChecked(isPilotEnabledForFacility =
```
Comment (qptr) [inline app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt:99]: shouldn't this parameter just be named `isCDSEnabledForFacility` now? If CDS is enabled for all facilities, there shouldn't be any need for pilots anymore.

Comment (qptr) [reply app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt:99]: Also, indentation seems off.

**Review by qptr — APPROVED** [review]

File: app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt (hunk 1 above)
Comment (siddh1004) [reply app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt:99]: @qptr This is not enabled for all facilities yet. Only enabled in ethiopia and selected facilities in other countries. Also, the indention is default for kotlin language and get set automatically.

Comment (qptr) [reply app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt:99]: Oh, does pilot mean CDS feature here?

Comment (siddh1004) [reply app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt:99]: yes

**Pull request merged**